runtime/src/heap_statistic.rs

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright 2022 Fluence Labs Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+use std::fmt;
+use std::ops::Deref;
+
+/// Contains module name and a size of its heap in bytes.
+#[derive(Debug)]
+pub struct HeapStatisticRecord<'module_name> {
+    pub module_name: &'module_name str,
+    pub memory_size: usize,
```

Comment:
is it memory or heap? Would be nice to have consistent naming across `TomlFaaSModuleConfig` and `HeapStatisticRecord`